compiler/rustc_incremental/src/assert_module_sources.rs

```diff
@@ -66,21 +67,17 @@ impl<'tcx> AssertModuleSource<'tcx> {
                 sym::post_dash_lto => (CguReuse::PostLto, ComparisonKind::Exact),
                 sym::any => (CguReuse::PreLto, ComparisonKind::AtLeast),
                 other => {
-                    self.tcx.sess.span_fatal(
-                        attr.span,
-                        &format!("unknown cgu-reuse-kind `{}` specified", other),
-                    );
+                    self.tcx
+                        .sess
+                        .emit_fatal(errors::UnknownReuseKind { span: attr.span, kind: other });
```

Comment:
The errors from this file are debugging tools for incremental tests. Do we really need to make them translatable? Do we expect non-english speakers to be able to contribute in english to the compiler?

Reply:
We don't need to, but I'd like to be able to remove the ability to write non-translatable diagnostics, just so that we can maintain fully translatable diagnostics once we reach that point.